game/views.py

```
@@ -4612,3 +4179,75 @@ def profile_view(request):
+            },
+            status=400,
+        )
+
+    if reply.is_deleted:
+        return JsonResponse(
+            {"success": False, "error": "Cannot vote on deleted replies."},
+            status=400
+        )
+
+    if vote_type == "up":
+        vote_value = ReplyVote.UPVOTE
```

Ruff (0.15.20)

[error] Undefined name `ReplyVote`

(F821)